### ISSUE TITLE
Hotkey helper dialog for adding new entries has unlabeled edit fields for screen reader users.

### ISSUE THREAD

**Issue by rkingett** [kunkel321/AutoCorrect2#4]
I tried to fix this one myself and submit a PR, but I was unable to find the edit fields in question in the script itself.

I am using the latest version of NVDA, a free and open source screen reader.

When adding a new entry with WINDOWS H, there are 3 edit fields that appear to not have labels on them.

The first one appears to be a prefix edit field? If I select a word or phrase or something, it gets populated with B0X.

The second is the word or phrase we want to correct. That one does not have an edit field labeled.

The third is the comments multiline edit box below the, make this a function, checkbox.

# Label proposals.

For the first one that gets prefixed with a B0X, I honestly don't know what that field is for so I can't suggest a label.

The second one, I'd suggest the lengthy label, "word or phrase you want to correct"

The third, "comments related to this autocorrect string"

I know edit fields aren't usually this long but I think it would be good for cognitive retention, too!

Again, thanks so much for your cooperation!

**Comment by kunkel321** [comment]
The first one is for the hotstring options, as described here: https://www.autohotkey.com/docs/v2/Hotstrings.htm#Options
It does have a label though.  The label is the text field above it that says "Options."
On about line 172 is: 
`hh.AddText('y4 w30', 'Options')`
which is the label.  Then the options edit box is three lines down
`MyDefaultOpts := hh.AddEdit(listColor ' yp+20 xm+2 w70 h24')`

Under each of those you'll see the trigger string label and edit box.  It's noteworthy that the trigger string label is dynamic and changes to alert the user if/when there are potential misspellings associated with the trigger.

A bit below that are the replacement string label
`hh.AddText('xm', 'Replacement')`
and edit box
`ReplaceString := hh.AddEdit(listColor ' +Wrap y+1 xs h' hFactor + 100 ' w' wFactor, '')`

I installed NVDA.  It's a pretty cool app, though the voice gets a bit annoying.  LOL.  I see what you mean about the edit boxes not getting verbally identified correctly.  I'm not sure how to embed that information into the code though... Is it possible?  I'll experiment more.

**Comment by rkingett** [comment]
Interesting! I actually wonder if this isn't an issue with AHK dialogs themselves and not so much this script? The replacements edit field reads just fine so I can figure out what the other 2 are, mostly, or I can just add strings manually but I thought I'd open a ticket anyway! After doing some digging I think I am gonna close this issue because it appears to be an issue with how AHK makes edit fields. Thanks for downloading it and giving it a try!

**Comment by kunkel321** [comment]
Yeah, It might be a limitation of AHK... I searched the help docs for "Accessibility" and got zero matches, LOL.  I noticed with the edit boxes in hotstring helper, NVDA mostly just said "Edit blank".  

It would be possible to have default text appear in the edit boxes, 
`ReplaceString := hh.AddEdit(listColor ' +Wrap y+1 xs h' hFactor + 100 ' w' wFactor, 'This would be the default text read be NVDA...')`
but that would be no good, because the contents of the boxes get populated by whatever text is selected when you press Win+H. 

I experimented with having hidden text controls near the edit boxes, but that didn't work.  NVDA won't read hidden text.
